kratos/utilities/read_materials_utility.cpp

```diff
@@ -131,10 +131,12 @@ void ReadMaterialsUtility::AssignPropertyBlock(Parameters Data)
 
     //Set the CONSTITUTIVE_LAW for the current p_properties.
     if (Data["Material"].Has("constitutive_law")) {
-        std::string constitutive_law_name = Data["Material"]["constitutive_law"]["name"].GetString();
+        Parameters cl_parameters = Data["Material"]["constitutive_law"];
+        std::string constitutive_law_name = cl_parameters["name"].GetString();
         TrimComponentName(constitutive_law_name);
+        cl_parameters["name"].SetString(constitutive_law_name);
 
-        auto p_constitutive_law = KratosComponents<ConstitutiveLaw>().Get(constitutive_law_name).Clone();
+        auto p_constitutive_law = KratosComponents<ConstitutiveLaw>().Get(constitutive_law_name).Create(cl_parameters);
```

Comment:
@josep-m-carbonell This only changed one line, not need to define the Create method